docs/family-pub-key.rst

```diff
@@ -0,0 +1,101 @@
+******************************
+Public Key Transaction Family
+******************************
+
+Overview
+========
+
+The *Public Key* transaction family provides a trustless way to store the public key based on RSA cryptosystem on the REMchain.
+
+Definitions of Public Key Entries
+---------------------------------
+
+The following protocol buffers definition define public key entries:
+
+.. code-block:: protobuf
+
+    message PubKeyStorage {
+         // The UID field of a certificate.
+         string owner = 1;
+         // indicates if this key was revoked
+         NewPubKeyPayload payload = 2;
+         bool revoked = 3;
+    }
+
+Addressing
+----------
+
+Address for storing the *PubKeyStorage* is constructed using *make_address_from_data* of the *BasicHandler* with a **public key** provided.
+
+Transaction Payload
+===================
+
+Public Key transaction family payloads are defined by the following protocol
+buffers code:
+
+.. code-block:: protobuf
+
+    message NewPubKeyPayload {
+        enum EntityType {
+            PERSONAL = 0;
+            SERVER = 1;
+        }
+
+        enum PubKeyType {
+            RSA = 0;
+        }
+        // PEM Encoded public key
+        string public_key = 1;
+        PubKeyType public_key_type = 2;
+        // For future ACME support(Now only Personal)
+        EntityType entity_type = 3;
+        // hash of PEM encoded public key of certificate
+        string entity_hash = 4;
+        string entity_hash_signature = 5;
+        // UNIX time when validity begins
+        uint32 valid_from = 6;
+        // UNIX time when validity ends
+        uint32 valid_to = 7;
+    }
+
+    message RevokePubKeyPayload {
+        // The address of a certificate in the storage generated by the transaction processor.
+        string address = 1;
+    }
+
+
+Transaction Header
+==================
+
+Inputs and Outputs
+------------------
+
+The inputs and outputs for account family transactions must include:
+
+* Sender's account address
```

Comment:
Whoah, what do we see here, a copy-paste ;)
Please fix that to the actual values.